infra/terraform/modules/data_access/team_events/requirements.dev.txt

```diff
@@ -0,0 +1,5 @@
+boto3
+
+pytest
+pytest-spec
+pytest-watch
```

Comment:
Correct, I'm not using any external module apart from `boto3` which is already available in AWS lambda. We could have a python virtualenv for the `data_access` module I guess.